documentation/modules/con-address-plans.adoc

```diff
@@ -47,7 +53,9 @@ A single router can support five instances of addresses and broker can support t
 In the `standard` address space, address plans for the `queue` address type may contain a field
 `partitions`, which allows a queue to be sharded across multiple brokers for HA and improved performance. Specifying an amount of `broker` resource above 1 will automatically cause a queue to be partitioned.
 
-include::ref-ttl.adoc[]
+include::ref-message-ttl.adoc[]
```

Comment:
I can't see this file because it was deleted.

Reply:
@hmaclean73 It has been moved and should be visible here: https://github.com/EnMasseProject/enmasse/blob/1351fc3fdee296fc129c24c2ea89c0e1512c77d8/documentation/modules/ref-message-ttl.adoc

Reply:
Thanks.